dango/types/src/lending.rs

```diff
@@ -0,0 +1,76 @@
+use {
+    grug::{Addr, Coins, Denom},
+    std::collections::BTreeSet,
+};
+
+/// The namespace that lending pool uses.
+pub const NAMESPACE: &str = "lending";
+
+#[grug::derive(Serde)]
+pub struct InstantiateMsg {
+    pub whitelisted_denoms: BTreeSet<Denom>,
+}
+
+#[grug::derive(Serde)]
+pub enum ExecuteMsg {
+    /// Whitelist a denom. Can only be invoked by the owner.
+    WhitelistDenom(Denom),
+
+    /// Delist a denom. Can only be invoked by the owner.
+    DelistDenom(Denom),
```

Comment:
In practice, we don't need to list or delist. Each denom is associated with an `InterestRateParams`. If a denom has such as params, it's listed; otherwise it's not listed.